File: Course 04 [Algorithms & Problem-Solving Level 1 Solutions]/My_Solutions_Using_C#/Problem9.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Net;
using System.Threading;
using System.Threading.Tasks;

namespace AbuHadHoud
{
    internal class Program
    {
        static void ReadNumber(out int Num1, out int Num2, out int Num3)
        {
            Console.Write("Please enter Number 1 : ");
            Num1 = int.Parse(Console.ReadLine());

            Console.Write("Please enter Number 2 : ");
            Num2 = int.Parse(Console.ReadLine());

            Console.Write("Please enter Number 3 : ");
            Num3 = int.Parse(Console.ReadLine());

        }

        static int SumOf3Numbers(int Num1, int Num2, int Num3)
        {
            return Num1 + Num2 + Num3;
        }
        static void PrintResult(int Total)
        {
            Console.WriteLine($"\nThe sum of Total numbers is  : {Total}");
        }
        static void Main(string[] args)

        {
            int N1, N2, N3;
            ReadNumber(out N1, out N2, out N3);
            PrintResult(SumOf3Numbers(N1,N2,N3));
        }
    }
}

File: Course 04 [Algorithms & Problem-Solving Level 1 Solutions]/Abu_HadHoud_Solutions_Using_C++/Problem9.cpp
#include <bits/stdc++.h>
using namespace std;

void ReadNumbers(int& Num1, int& Num2, int& Num3) {
	cout << "\nPlease enter your Number 1 : ";
	cin >> Num1;

	cout << "\nPlease enter your Number 2 : ";
	cin >> Num2;

	cout << "\nPlease enter your Number 3 : ";
	cin >> Num3;
}

int SumOf3Numbers(int Num1, int Num2, int Num3) {
	return Num1 + Num2 + Num3;
}

void PrintResult(int Total) {
	cout << "\nThe total sum of Numbers si  : " << Total;
}

int main() {
	int Num1, Num2, Num3;
	ReadNumbers(Num1, Num2, Num3);
	PrintResult(SumOf3Numbers(Num1, Num2, Num3));

}